crates/ruff_cli/src/args.rs

```diff
@@ -290,6 +290,10 @@ pub struct CheckCommand {
     /// The name of the file when passing it through stdin.
     #[arg(long, help_heading = "Miscellaneous")]
     pub stdin_filename: Option<PathBuf>,
+    /// List of mappings from file extension to language (one of ["python", "ipynb", "pyi"]). For
+    /// example, to treat `.ipy` files as IPython notebooks, use `--extension ipy:ipynb`.
+    #[arg(long, value_delimiter = ',', hide = true)]
```

Comment:
What's the reason for hiding the option? IMO, we should either add options if we feel confident that they're good or not add them, but adding them as hidden feels indecisive.

Reply:
Oops, I copied this over from the `linter` version -- I will remove both.